wiki/Beatmap/Packs/en.md

```diff
@@ -3,27 +3,40 @@ needs_cleanup: true
 outdated: true
 ---
 
-<!-- TODO: new pack types, new mirrors -->
+<!-- TODO: new mirrors -->
 
 # Beatmap packs
 
-A beatmaps pack is a `.zip` file containing beatmaps. The name of the pack is based on what they contain (e.g. `Approved Beatmap Pack #7` would only contain [approved](/wiki/Beatmap/Category#approved) beatmaps).
+A beatmaps pack is a compressed file containing beatmaps. It is worth noting that the most packs are:
 
-It is worth noting that the most packs are:
-
-- Roughly either ~100MB (Taiko/osu!mania) or ~200MB (-/Approved)
+- Roughly either ~100MB (Taiko/osu!mania) or ~200MB (-/Approved) <!-- TODO: Classifying like this looks inappropriate -->
```

Comment:
maybe @peppy or someone else could find an average and/or range of file sizes if that's relevant to mention